modules/installation-uninstall-infra.adoc

```diff
@@ -3,109 +3,21 @@
 // * installing/installing_osp/uninstalling-cluster-osp.adoc
```

Comment:
I think this assembly file path needs to be updated. `installing/installing_openstack/uninstalling-openstack-user.adoc`